### ISSUE TITLE
Not all options for osc getbinaries is listed 

### ISSUE THREAD

**Issue by lorddoskias** [openSUSE/open-build-service#4115]
Issue/Feature description
====================
When I run :` osc getbinaries` I get: 

```
Valid arguments for this package are:

SLE_12_SP3_Update  i586
SLE_12_SP3_Update  x86_64
SLE_12_SP2_Update  i586
SLE_12_SP2_Update  x86_64
SLE_12_SP2         i586
SLE_12_SP2         x86_64
```
However, the project also builds against SLE_12_SP3 and in fact running : `osc getbinaries SLE_12_SP3 x86_64` grabs the correct binaries. Here is the metadata of the configured repos: 

```
<project name="home:jeff_mahoney:lustre:2.10">
  <title>Lustre 2.10</title>
  <description/>
  <person userid="jeff_mahoney" role="maintainer"/>
  <repository name="SLE_12_SP3_Update">
    <path project="SUSE:SLE-12-SP3:Update" repository="standard"/>
    <arch>i586</arch>
    <arch>x86_64</arch>
  </repository>
  <repository name="SLE_12_SP3">
    <path project="SUSE:SLE-12-SP3:GA" repository="standard"/>
    <arch>i586</arch>
    <arch>x86_64</arch>
  </repository>
  <repository name="SLE_12_SP2_Update">
    <path project="SUSE:SLE-12-SP2:Update" repository="standard"/>
    <arch>i586</arch>
    <arch>x86_64</arch>
  </repository>
  <repository name="SLE_12_SP2">
    <path project="SUSE:SLE-12-SP2:GA" repository="standard"/>
    <arch>i586</arch>
    <arch>x86_64</arch>
  </repository>
</project>

```

Expected result
============
```
Valid arguments for this package are:

SLE_12_SP3_Update  i586
SLE_12_SP3_Update  x86_64
SLE_12_SP2_Update  i586
SLE_12_SP2_Update  x86_64
SLE_12_SP2         i586
SLE_12_SP2         x86_64
SLE_12_SP3         i586
SLE_12_SP3         x86_64
```

How to Reproduce
===============
1. Checkout `home:jeff_mahoney:lustre:2.10/lustre` project from Suse's internal build service
2. Run `osc getbinaries`

Further information
===============

This is observed on Suse's internal build service instance

**Comment by lethliel** [comment]
looks like an osc bug

**Comment by marcus-h** [comment]
On 2017-11-07 16:34:33 +0000, Marco Strigl wrote:
> looks like an osc bug
> 
Wild guess: the build is explicitly disabled in lustre's package
meta for the SLE_12_SP3 repo (maybe even for a different arch).

**Comment by lethliel** [comment]
On 11/07/2017 06:19 PM, Marcus Hüwe wrote:> Wild guess: the build is explicitly disabled in lustre's package
> meta for the SLE_12_SP3 repo (maybe even for a different arch).
> 

That was my first thought too. The build really gets disabled for SLE_12_SP3 in the 
package meta. But not for x86_64. So IMHO SLE_12_SP3 x86_64 should be listed in
osc getbinaries. 

But just had the chance to take a very short look into this. I copied the project 
and will take a look tomorrow.
